client/web/src/components/widgets/inputs/MenuBarInput.vue

```diff
@@ -126,6 +126,10 @@ const menuEntries: MenuListEntries = [
 			[
 				{ label: "Select All", shortcut: ["Ctrl", "A"], action: async () => (await wasm).select_all_layers() },
 				{ label: "Deselect All", shortcut: ["Ctrl", "Alt", "A"], action: async () => (await wasm).deselect_all_layers() },
+				{ label: "Move Layer Up", shortcut: ["Ctrl", "]"], action: async () => (await wasm).move_selected_layer(1) },
+				{ label: "Move Layer Down", shortcut: ["Ctrl", "["], action: async () => (await wasm).move_selected_layer(-1) },
+				{ label: "Move Layer To Top", shortcut: ["Ctrl", "Shift", "]"], action: async () => (await wasm).move_selected_layer(999999) },
+				{ label: "Move Layer To Bottom", shortcut: ["Ctrl", "Shift", "["], action: async () => (await wasm).move_selected_layer(-999999) },
```

Comment:
Let's put these in a flyout menu category called "Arrange" and use the following naming:
"Raise"

"Lower"

Reply:
Sure thing. Could you clarify exactly what you mean by "flyout menu" though? Is there precedence for this type of feature elsewhere in the codebase? My skills lie much more in backend than frontend so it might make more sense for someone else to hook it up if it's not trivial.

Reply:
A sub-menu. See "Open Recent" a few lines above for an example.

Reply:
I've set this up but the menu items now no longer their actions, I think there's a bug somewhere in `MenuList.vue`. I'd guess grandchildren's actions are not being registered, but I'm not familiar enough with Vue to find a solution.

Reply:
Working on fixing this now, it appears that the click handler for submenu entries isn't being called for some reason. You are fine to commit this but my fix should solve the functionality aspect.